lib/npc-input.js

```diff
@@ -315,11 +315,7 @@ export class NpcInput extends FormApplication {
     ts.title = m.title
     let dt = new Date().toString().split(' ').splice(1, 3).join(' ')
     ts.createdon = dt
-    if (!!m.sm)
-      ts.sizemod =
-        m.sm[0] == '-' ? m.sm
-        : m.sm[0] == '+' ? ''
-        : '+' + m.sm
+    if (!!m.sm) ts.sizemod = m.sm[0] == '-' ? m.sm : m.sm[0] == '+' ? '' : '+' + m.sm
```

Comment:
this was just done automatically by prettier according to our current config, so I'm leaving it in unless it's a problem.